src/course-outline/card-header/CardHeader.jsx

```diff
@@ -61,6 +61,13 @@ const CardHeader = ({
   const [titleValue, setTitleValue] = useState(title);
   const cardHeaderRef = useRef(null);
   const [isManageTagsDrawerOpen, openManageTagsDrawer, closeManageTagsDrawer] = useToggle(false);
+  let fullProctoringExamConfigurationLink = proctoringExamConfigurationLink;
+
+  if (proctoringExamConfigurationLink && !(
+    proctoringExamConfigurationLink.indexOf('http://') === 0 || proctoringExamConfigurationLink.indexOf('https://') === 0
+  )) {
+    fullProctoringExamConfigurationLink = new URL(proctoringExamConfigurationLink, getConfig().STUDIO_BASE_URL).href;
```

Comment:
```suggestion
    fullProctoringExamConfigurationLink =  () => new URL(proctoringExamConfigurationLink, getConfig().STUDIO_BASE_URL).href;
```

Reply:
There are known issues with Tutor where the variable will not generate the link unless `new URL` is called as the return of a function.

Reply:
@KristinAoki Thanks! updated.